src/components/mainPatient/mainPatient.js

```diff
@@ -0,0 +1,44 @@
+import { useState } from 'react'
+import { useWindowWidth } from '@react-hook/window-size'
+
+import {
+  MainWrapper,
+  ButtonWrapper,
+  Button,
+  FilterWrapper,
+  FilterButton,
+} from '../mainDoctor/styles'
+
+import Title from '../title'
+import AppointmentsList from './appointmentsList'
+import EmptyState from '../mainDoctor/emptyWrapper'
+
+import data from './db.json'
+
+const MainPatient = () => {
+  const [state, setstate] = useState([data])
+  const width = useWindowWidth()
+
+  return (
+    <MainWrapper>
+      <ButtonWrapper>
+        <Button white>Profile</Button>
+        <Button>Appointments</Button>
+        {width >= 1280 && (
```

Comment:
Kinda wierd practice, to use it in template, probably you can resolve this using STYLES.

Reply:
I will use styles then